Formula/suite-sparse.rb

```diff
@@ -0,0 +1,24 @@
+class SuiteSparse < Formula
+  desc "Suite of Sparse Matrix Software"
+  homepage "http://faculty.cse.tamu.edu/davis/suitesparse.html"
+  url "http://faculty.cse.tamu.edu/davis/SuiteSparse/SuiteSparse-4.5.5.tar.gz"
+  sha256 "b9a98de0ddafe7659adffad8a58ca3911c1afa8b509355e7aa58b02feb35d9b6"
+
+  depends_on "metis"
+
+  def install
+    system "make", "library", "install", "INSTALL=#{prefix}",
+                   "BLAS=-framework Accelerate", "LAPACK=$(BLAS)",
+                   "MY_METIS_LIB=-L#{Formula["metis"].opt_lib} -lmetis",
+                   "MY_METIS_INC=#{Formula["metis"].opt_include}"
+
+    # Install a simple test
```

Comment:
The comment probably isn't needed at all, but if you do want to keep it, it'll fit inline.
```
    pkgshare.install "KLU/Demo/klu_simple.c" # install a simple test 
```